docs/self-managed/platform-deployment/helm-kubernetes/guides/connecting-to-entra-id.md

```diff
@@ -0,0 +1,45 @@
+---
+id: connecting-to-entra-id
+title: "Connecting to Entra ID (Azure AD)"
+description: "Learn how to connect your Camunda Self-Managed deployment to Entra ID (Azure AD)."
+---
+
+:::note
+The following components support Entra ID authentication: Operate
+:::
+
+This guide will walk you through the steps to connect your Camunda Self-Managed deployment to Entra ID (Azure AD).
+
+## Prerequisites
+
+- Ability to create applications in Entra ID
+
+:::note
+You should use the following steps to create an application in Entra ID for each of the components you want to connect
+to Entra ID.
+:::
+
+## Steps
+
+1. Access the Entra ID admin area
+   and [register an application](https://learn.microsoft.com/en-us/entra/identity-platform/quickstart-register-app).
+   After registering the app, the **Overview** page will contain a Client ID; make a note of this value as it will be
+   required later on.
+
+2. Within the app registered in step
+   1, [configure a platform](https://learn.microsoft.com/en-gb/entra/identity-platform/quickstart-register-app#configure-platform-settings)
+   of type `web`. The expected redirect URI of the component you are configuring an app for can be found in [component-specific configuration](#component-specific-configuration).
+
+3. Once you have registered a platform for your app a [client secret should be added](https://learn.microsoft.com/en-us/entra/identity-platform/quickstart-register-app#add-a-client-secret). Make a note of the value of the client secret as it will be required later on.
```

Comment:
client secret **should** be added or **will** be added?

Reply:
Ah I've confused the language here I think! What needs to happen is the customer needs to go and create a secret for the app, so maybe this should be reworded to something like:

```
Once you have registered a platform for your app a client secret needs to be created, to do this see [adding a client secret](https://learn.microsoft.com/en-us/entra/identity-platform/quickstart-register-app#add-a-client-secret)
```

wdyt?

Reply:
Just needs a period, then good to go!